ingest/ingest_pipeline.py

```diff
@@ -427,6 +445,16 @@ def subsample(self):
                         if load_status != 0:
                             return load_status
                 else:
+                    # Caution: recording errorTypes in this manner can clobber other collected errors.
+                    # In subsampling, known failure modes are ValueErrors which stop processing so
+                    # this logging approach should not lose file validation information
+                    config.get_metric_properties().update(
+                        {
+                            "errorTypes": ["content:missing:values-across-files"],
+                            "numErrortypes": 1,
```

Comment:
Blocker: this trivial casing bug is worth fixing now, even if deferring the `set_mixpanel_num` fix described in my other comment:

```suggestion
                            "numErrorTypes": 1,
```

Reply:
removed `"numErrortypes": 1`, should be handled by the fix in `set_mixpanel_num` in `45230a2`